core/src/main/java/org/frankframework/jdbc/JdbcQuerySenderBase.java

```diff
@@ -152,7 +151,13 @@ public enum QueryType {
 		/** To execute Oracle PL/SQL package */
 		PACKAGE,
```

Comment:
Ik denk dat `PACKAGE`, `UPDATECLOB` en `UPDATEBLOB` ook wel mogen worden gedeprecate.

`PACKAGE` in ieder geval, is al heel lang niet eens getest door iemand. @nielsm5 Wat denk jij van `UPDATECLOB` en `UPDATEBLOB`? Die zijn toch niet meer nodig ondertussen?

Reply:
Voor nu valt dit even buiten mijn scope van 'even snel de laatste enum-method-name-suffixes' weghalen.